cotk/scripts/report.py

```diff
@@ -150,27 +151,45 @@ def upload_report(result_path, entry, args, \
 	LOGGER.info("Save your report locally at .cotk_upload_backup")
 	json.dump(upload_information, open(".cotk_upload_backup", 'w'))
 	LOGGER.info("Uploading your report...")
-	res = requests.post(REPORT_URL, upload_information)
+	res = requests.post(REPORT_URL, {"data": upload_information, "token": token})
 	res = json.loads(res.text)
 	if res['code'] != "ok":
 		raise RuntimeError("upload error. %s" % json.loads(res['err']))
 	return res['id']
 
+def get_local_token():
+	'''Read locally-saved token'''
+	if os.path.exists('.cotk_config'):
+		return json.load(open('.cotk_config', 'r'))['token']
+	else:
+		raise RuntimeError("Please specify a token.")
+
+def save_token(token):
+	'''Save token locally'''
+	json.dump({'token': token}, open(".cotk_config", "w"))
+	LOGGER.info("Save your configuration locally at .cotk_config")
+
 def report(args):
 	'''Entrance of report'''
-	parser = argparse.ArgumentParser(prog="cotk-report", \
+	parser = argparse.ArgumentParser(prog="cotk report", \
 		description='Report model performance to cotk model dashboard.')
-	parser.add_argument('--result', type=str, default="result.json", \
+	parser.add_argument('--token', type=str, default="")
```

Comment:
Use `cotk config token` to set token,  report with token should not change the config, but used temporarily